python/pyspark/sql/pandas/types.py

```diff
@@ -162,6 +243,8 @@ def from_arrow_type(at: "pa.DataType", prefer_timestamp_ntz: bool = False) -> Da
         spark_type = StringType()
     elif types.is_binary(at):
         spark_type = BinaryType()
+    elif types.is_fixed_size_binary(at):
+        spark_type = BinaryType()
```

Comment:
is it related?

Reply:
It's not required. I will move this to a separate PR.

Reply:
Moved to #46688